docs/app/internationalization.md

```diff
@@ -29,3 +29,45 @@ Locale messages should only ever be loaded on the server-side, to avoid bloating
 1. Add a language folder, using the same BCP47 language tag: `mkdir -p src/i18n/messages/<lang>`
 1. Add a language file: `touch src/i18n/messages/<lang>/index.ts` and add the translated content. The JSON structure should be the same across languages. However, non-default languages can omit keys, in which case the default language will be used as a fallback.
 1. Update [`i18n/config.ts`](../app/src/i18n/config.ts) to include the new language in the `locales` array.
+
+## Structuring your messages
+
+In terms of best practices, [it is recommended](https://next-intl-docs.vercel.app/docs/usage/messages#structuring-messages) to structure your messages such that they correspond to the component that will be using them. You can nest these definitions arbitrarily deep if you have a particularly complex need.
+
+It is always preferable to structure messages per their usage rather than due to some side effect of a technological implementation.
```

Comment:
Is there a good vs bad example to help further convey this point?

Reply:
I added further clarification in text... Do you think that's sufficient, or should we add a good/bad code block?

Reply:
The example you added about splitting up a paragraph is helpful, thanks!